Document issue where `,` can't find previous match (#38)
VSpaceCode/vspacecode.github.io: docs/troubleshooting.md

],
```

## `,` cannot find previous match

> When the key `,` cannot get back to the previous match after a find `f{character}` with Vim.
> This is because the default VSpaceCode configuration overrides `,` as a shortcut for the major mode.
> You can remove follow settings in your `settings.json` to remove this shortcut.

Reviewer: ```suggestion
The default VSpaceCode configuration overrides `,` as a shortcut for the [major mode](./major-mode.md),
therefore the key `,` cannot get back to the previous match after a find `f{character}` with Vim.

In order to remove this major mode shortcut, remove the following section from your `settings.json`:
```

```json
{
    "before": [","],
    "commands": [
        "vspacecode.space",
        {
            "command": "whichkey.triggerKey",
            "args": "m"
        }
    ],
}
```

> You can also change the shortcut key by changing `","` in `"before"` to other keys.

Reviewer: ```suggestion
Alternatively, you can change the shortcut key by changing `","` in `"before"` to other keys.
```
